### PR TITLE
refine error handling for existing worksheet

### PR DESCRIPTION
**Pull request by rioliu-rh**
```
$ oar -r 4.12.62 create-test-report
2024-08-09T18:46:55Z: INFO: test report of 4.12.62 already exists, url: xxx
```

### CODE REVIEW

File: oar/core/exceptions.py [hunk 1]
```diff
@@ -6,17 +6,25 @@ class WorksheetException(BaseException):
     """Exception class to raise error in WorksheetManager"""
 
 
+class WorksheetExistsException(WorksheetException):
+    """Exception class to raise workhseet already exists error"""
```
Comment (tomasdavidorg) [inline oar/core/exceptions.py:10]:
```suggestion
    """Exception class to raise worksheet already exists error"""
```

File: oar/core/exceptions.py [hunk 2]
```diff
@@ -6,17 +6,25 @@ class WorksheetException(BaseException):
...
+
+
```
Comment (tomasdavidorg) [inline oar/core/exceptions.py:12]: Are the new lines intended? There are two now between the classes.

**Review by tomasdavidorg — COMMENTED** [review]
Just small typo.

File: oar/core/exceptions.py (hunk 2 above)
Comment (rioliu-rh) [reply oar/core/exceptions.py:12]: it is formatted by plugin `"editor.defaultFormatter": "ms-python.black-formatter"`

Comment (rioliu-rh) [reply oar/core/exceptions.py:12]:
```
"[python]": {
    "diffEditor.ignoreTrimWhitespace": false,
    "gitlens.codeLens.symbolScopes": ["!Module"],
    "editor.defaultFormatter": "ms-python.black-formatter",
    "editor.formatOnSave": true,
    "editor.autoClosingBrackets": "always"
  }
```
this is my vscode config for python code formatter

Comment (tomasdavidorg) [reply oar/core/exceptions.py:12]: Ok, thanks 🙂. I was not sure if it is a auto-formatter or a typo.

**Pull request merged**